src/cargo/ops/registry.rs

```diff
@@ -34,6 +34,11 @@ pub fn publish(manifest_path: &Path,
                verify: bool) -> CargoResult<()> {
     let pkg = try!(Package::for_path(&manifest_path, config));
 
+    if *pkg.private() {
+        bail!("private crates cannot be published to crates.io.\n\
+        `{}` is a private crate", pkg.name());
```

Comment:
For formatting this, could the second line be lined up with the `p` in `private`?